packit_service/worker/handlers/forges_handlers.py

```diff
@@ -384,9 +384,9 @@ def pre_check(self) -> bool:
         return True
 
 
-@reacts_to(event=PullRequestLabelPagureEvent)
-class PagurePullRequestLabelHandler(JobHandler):
-    task_name = TaskName.pagure_pr_label
+@reacts_to(MergeRequestLabelGitlabEvent)
+class PullRequestLabelHandler(JobHandler):
+    task_name = TaskName.pr_label
```

Comment:
Do we want to preserve the old functionality? (=Do we want to remove the old `reacts_to` decorator?)

Reply:
My thinking is that this create-bug-from-mr feature is tightly coupled with c8s source-git 2 dist-git workflow.
Since we've already moved c8s source-git repos from CentOS Pagure to Gitlab, there's in my POV, minimal chance that we will ever need this also for Pagure.
And I'm a fan of cleaning unused code at the moment I stumble upon it as one can always dig it from history.

Reply:
Yes, I agree. I just wanted to be sure that it's what you want and that we can support multiple events.